src/main/java/seedu/address/logic/AddressbookLogic.java

```diff
@@ -4,13 +4,12 @@
 import seedu.address.logic.commands.CommandResult;
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.anakindeck.AnakinCard;
-import seedu.address.model.anakindeck.AnakinDeck;
+import seedu.address.model.person.Person;
 
 /**
- * API of the Logic component
+ * API of the AddressbookLogic component
  */
-public interface AnakinLogic {
+public interface AddressbookLogic {
```

Comment:
Looks like the anakin specific functions have been ported over to `Logic` already. We should be able to delete this file?

Reply:
I leave the delete part until we finish integration